compiler/tf2tfliteV2/tf2tfliteV2.py

```diff
@@ -114,10 +115,32 @@ def _get_parser():
     return parser
 
 
+def _apply_verbosity(verbosity):
+    # NOTE
+    # TF_CPP_MIN_LOG_LEVEL
+    #   0 : INFO + WARNING + ERROR + FATAL
+    #   1 : WARNING + ERROR + FATAL
+    #   2 : ERROR + FATAL
+    #   3 : FATAL
+    #
+    # TODO Find better way to suppress trackback on error
+    # tracebacklimit
+    #   The default is 1000.
+    #   When set to 0 or less, all traceback information is suppressed
+    if verbosity:
+        os.environ['TF_CPP_MIN_LOG_LEVEL'] = '0'
+        sys.tracebacklimit = 1000
+    else:
+        os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+        sys.tracebacklimit = 0
+
+
 def _check_flags(flags):
     """
   Checks the parsed flags to ensure they are valid.
   """
+    _apply_verbosity(flags.verbose)
```

Comment:
How about invoke this function at the first of `_convert`?
```python
def _convert(flags):
    _apply_verbosity(flags.verbose)
    if (flags.v1):
        _v1_convert(flags)
    else:
        _v2_convert(flags)
```

Reply:
Okay, I will update.